.github/workflows/test-on-centos.yml

```diff
@@ -0,0 +1,42 @@
+name: Run tests in Centos container
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 1 * * *'
+
+jobs:
+  tests-on-centos:
+    strategy:
+      matrix:
+        version: ["c8s", "c9s"]
+    name: "Unittests on Centos ${{ matrix.version }}"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v4
+      - name: "Run in container"
+        uses: addnab/docker-run-action@v3
```

Comment:
Any reason to not use the same thing as in https://github.com/osbuild/osbuild/blob/345516e86748fc586d451bd5046aa13eddd757ca/.github/workflows/test.yml#L40C55-L40C95 ?

It seems to do much more than your code. E.g. mounting additional directories to the container.

Reply:
Hmmm, this is a good question. When I checked it in the 1st iteration I had an impression that `osbuild/containers/src/actions/privdocker@552e30cf1b4ed19c6ddaa57f96c342b3dff4227b` is deprecated and we should stick to upstream actions. @ochosi I am not sure but I was asking you about difference between our "internal" actions and upstream ones long time ago. Which is a better fit here?